ha/setup/create_pacemaker_resources.py

```diff
@@ -58,8 +59,10 @@ def cib_get(cib_xml):
     process.run_cmd(f"pcs cluster cib {cib_xml}")
     return cib_xml
 
-def hax(cib_xml, push=False, **kwargs):
+def hax(cib_xml, create=False, push=False, **kwargs):
```

Comment:
What is the idea behind this introduced logic?

Reply:
This is used to create separate resource
we need to create mgmt resource during adding 2nd node so to keep same with mgmt vip implementation added this.
Step and call still same for all except mgmt vip where create is true

Reply:
why can't vip be created on first node itself?

Reply:
During this we are not sure  is it single node or multi-node configuration
So calling mgmt_vip() after node add and count == 2

Reply:
Number of nodes can be obtained from provisioner confstore.

Reply:
Yes there is way,

`len(Conf.get(self._index, f"server_node"))`

Do we need to change template for this as template only have single node in file for 3 node also ?

We can get node count by len(Conf()) but i think there was discussion Confstore is key val only and getting top element and then traverse may not allowed 
(currently i am able to get count as 3 so it is possible to implement it)

@ajaysrivas @indrajitzagade

Reply:
Updated code

Reply:
Code is updated removed create=False as no longer needed